app/components/activity/ActivityDetails.js

```diff
@@ -0,0 +1,104 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+	Text,
+	View,
+  Button,
+} from 'react-native';
+import ActivityStyles from '../../styles/ActivityStyles';
+import Header from '../Header'
+import ListSlider from '../ListSlider';
+
+class ActivityDetails extends React.Component {
+	static navigationOptions = ({ navigation }) => ({
+		header: (
+			<Header
+				title="Activity Details"
+				hasBackButton
```

Comment:
could you make this hasBackButton = { true }

Reply:
@pratyushpal she's correct to have it as `hasBackButton`.  We can do this if it is a boolean.